packages/bsky/tests/views/__snapshots__/timeline.test.ts.snap

```diff
@@ -2898,22 +2898,22 @@ Array [
                 "images": Array [
                   Object {
                     "alt": "tests/sample-img/key-landscape-small.jpg",
-                    "fullsize": "https://bsky.public.url/img/feed_fullsize/plain/user(5)/cids(2)@jpeg",
-                    "thumb": "https://bsky.public.url/img/feed_thumbnail/plain/user(5)/cids(2)@jpeg",
+                    "fullsize": "https://bsky.public.url/img/feed_fullsize/plain/user(3)/cids(2)@jpeg",
```

Comment:
These are all just orderings of dids/cids cause of json key ordering. Figured it was easier to update snaps than figure out where exactly it was misordered